### PR TITLE
storagenode/orders/ordersfile: unit test coverage 

### PR DESCRIPTION
**Pull request by storj-antonio**
What: 

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

**Review by ihaid — APPROVED** [review]

**Review by profclems — APPROVED** [review]

File: storagenode/orders/ordersfile/common_test.go [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+	var tests []struct {
+		name string
+		args func(t *testing.T) args
+
+		want1      Writable
+		wantErr    bool
+		inspectErr func(err error, t *testing.T)
+	}
+
+	for _, tt := range tests {
```
Comment (egonelbre) [inline storagenode/orders/ordersfile/common_test.go:33]: There are no tests begin run here.

File: storagenode/orders/ordersfile/common_test.go [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+			}
+		})
+	}
+}
+
+func TestGetUnsentInfo(t *testing.T) {
+	type args struct {
+		info os.FileInfo
+	}
+	var tests []struct {
```
Comment (egonelbre) [inline storagenode/orders/ordersfile/common_test.go:55]: There are no tests begin run here.

File: storagenode/orders/ordersfile/common_test.go [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+			}
+		})
+	}
+}
+
+func TestGetArchivedInfo(t *testing.T) {
+	type args struct {
+		info os.FileInfo
+	}
+	var tests []struct {
```
Comment (egonelbre) [inline storagenode/orders/ordersfile/common_test.go:86]: There are no tests begin run here.

File: storagenode/orders/ordersfile/common_test.go [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+		})
+	}
+}
+
+func TestOpenReadable(t *testing.T) {
+	type args struct {
+		path    string
+		version Version
+	}
+	var tests []struct {
```
Comment (egonelbre) [inline storagenode/orders/ordersfile/common_test.go:118]: There are no tests begin run here.

**Pull request merged**